documentation/specs/proposed/BuildCheck.md

```diff
@@ -75,14 +75,12 @@ Majority of following cases are included in appropriate context within the scena
 ```ini
 # I expect this to apply to all projects within my solution, but not to projects which are not part of the solution
 [ContosoFrontEnd.sln]
-build_check.BC0101.IsEnabled=true
 build_check.BC0101.Severity=warning
```

Comment:
nit: while you are in this file, can you unify the build check prefix used in this doc? both `build_check` and `buildcheck` are here and it is very confusing to know which is actually in use.

Reply:
Didn't notice it at first, thank you! One inconsistency aligned now :)